docs/topics/ksp/ksp-quickstart.md

```diff
@@ -1,12 +1,15 @@
 [//]: # (title: KSP quickstart)
 
-For a quick start, you can create your own processor or get a [sample one](https://github.com/google/ksp/tree/main/examples/playground).
+For a quick start, you can create your own processor or get a [sample one](https://github.com/google/ksp/releases/download/1.5.30-1.0.0/playground.zip).
```

Comment:
Oops, I missed the comment in the above. The sample dir is maintained to match the latest release instead of master/main branch. I think it is fine to keep using it?

Reply:
Indeed, I messed something up here. [Fixed.](https://github.com/JetBrains/kotlin-web-site/pull/2716/commits/fef66bb725235700a2f84b84b6a86a756d9ee346)